src/lib/components/composites/settings/project-settings/slr/FetcherOptionsDialog.svelte

```diff
@@ -0,0 +1,156 @@
+
+    interface Option {
+        name: string;
+        value: string;
+        defaultValue: string;
+    }
+
+    let options: Map<string, Option> = $state(new Map());
+
+    $effect(() => {
+        if (fetcher !== "" && open) refetchOptions(fetcher);
+    });
+
+    // Fetch the available options for a fetcher
+    async function refetchOptions(fetcher: string) {
+        loading = true;
+        const availableOptions = await backendService
+            .getAvailableFetcherOptions({ fetcherName: fetcher })
+            .then((it) => new Map(Object.entries(it.response.options)));
+        const fetchers = new Map(Object.entries(projectSettings?.fetchers ?? []));
+        const fetcherOptions = new Map(Object.entries(fetchers.get(fetcher)?.options ?? {}));
+
+        options = new Map(
+            availableOptions.entries().map(([name, defaultValue]): [string, Option] => [
+                name,
+                {
+                    name,
+                    value: fetcherOptions.get(name) ?? "",
+                    defaultValue,
+                },
+            ]),
+        );
+        loading = false;
+    }
+
+    // Save fetcher options (options) of the project (projectId)
+    async function saveFetcherOptions() {
+        const updatedFetcherApis = projectSettings?.fetchers ?? {};
+        updatedFetcherApis[fetcher] = {
+            options: Object.fromEntries(
+                options
+                    .values()
+                    .filter((it) => it.value !== "")
+                    .map((it) => [it.name, it.value]),
+            ),
+        };
+
+        updateFetchers(
+            projectId,
+            updatedFetcherApis,
+            (it) => {
+                open = false;
+                onProjectChanged(it);
+            },
+            (it) => (error = it),
+        );
+    }
+
+    let loading = $state(false);
+
+    const headers: [string, string][] = [
+        ["Overridden", "Indicates wether the fetcher receives this option."],
+        ["Name", "The name of the option."],
+        ["Value", "The value of the option."],
+        ["Default", "Set the current value to the default value of the option."],
+    ];
+</script>
+
+<AlertDialog
+    actionButtonText="Save"
+    actionProps={{
+        onclick: () => saveFetcherOptions(),
+    }}
+    cancelButtonText="Cancel"
+    cancelProps={{
+        onclick: () => {},
+    }}
+    title="Edit Option Values"
+    bind:open
+    bind:loading
+>
+    {#snippet description()}
+        <div class="flex flex-col gap-4">
+            {#if loading}
+                {#each [0, 1, 2, 3] as i (i)}
+                    <div class="flex flex-row gap-2">
+                        <Skeleton class="h-8 w-8" />
+                        <Skeleton class="h-8 flex-1" />
+                        <Skeleton class="h-8 flex-1" />
+                        <Skeleton class="h-8 w-8" />
+                    </div>
+                {/each}
+            {:else if options.size == 0}
```

Comment:
Use `===` instead of `==`.
```suggestion
            {:else if options.size === 0}
```